easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
@stdweird: your suggestion will fix the unformatted raised error message, but it will not fix the raising that's not expected by gc3pie (cc @riccardomurri)